roles/confluent.kafka_broker/tasks/health_check.yml

```diff
@@ -62,7 +62,7 @@
 
 - name: Wait for Metadata Service to start
   uri:
-    url: "{{mds_http_protocol}}://{{ mds_advertised_listener_hostname | default(inventory_hostname) }}:{{mds_port}}/security/1.0/authenticate"
+    url: "{{mds_http_protocol}}://{{ mds_advertised_listener_hostname | default(inventory_hostname, True) }}:{{mds_port}}/security/1.0/authenticate"
```

Comment:
Can you explain the logic on this?  Why do we need the 'true' value here?  Do we need to set something to false if we don't want the inventory name?

Reply:
This variable was not declared in the default vars. Now since we declared with default to en empty string, is not more a null value. to over come we have a few ways - 
1.  The changes this PR has which evaluates if the string is none empty 
2. check the length of the variable. 
3. Define another variable to and set to false. 
I guess you are suggesting to last option. I felt that will change the way its getting used. So, I'm kind of inclined to use either 1 or 2. 
Let me know your opinion.